src/benchmarks/micro/libraries/System.Formats.Tar/Perf.TarFile.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+
+namespace System.Formats.Tar.Tests
+{
+    [BenchmarkCategory(Categories.Libraries)]
+    public class Perf_TarFile
+    {
+        /*
+        Always exist:
+            root/
+                inputdir/
+                        testdir/
+                        file.txt
+
+        Created on demand:
+                outputdir/
+                output.tar
+        */
+        private static readonly string _rootDirPath = FileUtils.GetTestFilePath();
+        private static readonly string _inputDirPath = Path.Combine(_rootDirPath, "inputdir");
+        private static readonly string _outputDirPath = Path.Combine(_rootDirPath, "outputdir");
+        private static readonly string _testDirPath = Path.Combine(_inputDirPath, "testdir");
+        private static readonly string _testFilePath = Path.Combine(_inputDirPath, "file.txt");
+        private static readonly string _inputTarFilePath = Path.Combine(_rootDirPath, "input.tar");
+        private static readonly string _outputTarFilePath = Path.Combine(_rootDirPath, "output.tar");
+        private MemoryStream _memoryStream = null;
+
+
+        // Setup and Cleanup
+
+        [GlobalSetup]
+        public void Setup()
+        {
+            Directory.CreateDirectory(_rootDirPath);
+            Directory.CreateDirectory(_inputDirPath);
+            Directory.CreateDirectory(_testDirPath);
```

Comment:
I have learned this very recently, but when you want to create `./a/b/c` and `./a` does not exist, then `Directory.CreateDirectory` is going to create it for you (same goes for `./a/b` and `./a/b/c`)

```suggestion
            Directory.CreateDirectory(_testDirPath);
```

Reply:
You're right, it's in the summary of that API 😄 : `Creates all directories and subdirectories in the specified path unless they already exist.`